content/02.introduction.md

```diff
@@ -10,66 +10,62 @@ One week later, 4593 confirmed cases had been identified, spanning not only Asia
 At this time, over 92,000 deaths had been reported due to COVID-19 worldwide.
+New information is emerging daily about the impact of the disease around the world.
 
-[Note: Maybe add a graph here, update as new reports come out.]
-
-COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
-SARS-CoV-2 is a coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
+COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was initially called 2019 novel Coronavirus (19-nCov).
+SARS-CoV-2 belongs to coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
 Infectious diseases of global concern have previously been associated with coronaviruses, including Severe Acute Respiratory Syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9]; however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS).
 Additionally, there are four endemic human coronaviruses that rarely progress beyond the mild symptoms associated with the common cold [@doi:10.1038/s41579-018-0118-9].
 The precise identity of SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
 As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, information about the transmission and life cycle of the virus as well as the diagnosis and treatment of the disease is emerging rapidly.
 In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
 This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.
 
-### About Coronaviruses
+### 2.2 Coronaviruses
 
-#### Classification:
+#### 2.2.a Classification
 Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses [@isbn:0781760607]. 
-Members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species, as well as SARS-CoV-2 [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].
+Members of this family include the species Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV) [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].
```

Comment:
```suggestion
Members of this family include the species Middle East respiratory syndrome-related coronavirus (MERS-CoV) and severe acute respiratory syndrome coronavirus (SARS-CoV) [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].
```

Reply:
I don't think you have to redefine this acronym. It's earlier in the intro :)

Reply:
Agreed..didn't realize that when I wrote it initially!